java/arcs/android/common/resurrection/ResurrectorService.kt

```diff
@@ -39,7 +39,9 @@ abstract class ResurrectorService : Service() {
      */
     protected open val resurrectionDatabaseName: String = DbHelper.RESURRECTION_DB_NAME
 
-    private val job = Job() + Dispatchers.IO + CoroutineName("ResurrectorService")
+    protected open val job =
+        Job() + Dispatchers.IO + CoroutineName("ResurrectorService")
+
```

Comment:
keep in mind if you override it in a subclass, but refer to it during constructor initialization in ResurrectorService, you can end up reading the wrong value. However, its safe because no work is done with job until onStartCommand in this class.

Reply:
I think you can't construct a Service yourself in Android as the system does it for you so we are safe. Or you were referring to some other case?